DataManagementSystem/Client/DataManager.py

```diff
@@ -190,7 +190,7 @@ def __getCatalogDirectoryContents( self, directories ):
     allFiles = {}
     while len( activeDirs ) > 0:
       currentDir = activeDirs[0]
-      res = returnSingleResult( self.fc.listDirectory( currentDir ) )
+      res = returnSingleResult( self.fc.listDirectory( currentDir, verbose = True ) )
```

Comment:
I always found this "feature" insane (sorry). Using a "verbose" keyed argument for defining what the return value is is really misleading... In addition the output structure of a method should always be the same IMHO